### PR TITLE
Update changelog for 4.7.0

### PR DESCRIPTION
**Pull request by trwalke**

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,3 +1,8 @@
+4.7.0
+============
+New Features:
+Added Subject Name + Issuer authentication to the acquire token by authorization code and acquire token by refresh token flows with the WithSendX5C() api on the confidential client application.
+
```
Comment (jennyf19) [inline changelog.txt]: nit: can you set it up like the other notes, w/a similar format, we have issue in bold and customer impact after that and then a link to the issue. thx.

File: changelog.txt [hunk 2]
```diff
@@ -1,3 +1,8 @@
+4.7.0
+============
+New Features:
+**Added Subject Name + Issuer authentication to the acquire token by authorization code and acquire token by refresh token flows with the WithSendX5C() api on the confidential client application.** All confidential client authentication flows will not have acces to this feature. [Issue](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1490)
```
Comment (jennyf19) [inline changelog.txt]: "will now have access" ? or not have access? also add another "s" to access :) thx.  looks cool

Comment (trwalke) [reply changelog.txt]: should be "will now have access"

Comment (jennyf19) [reply changelog.txt]: okay..because it says "will not"

**Pull request merged**